stubs/controller.py

```diff
@@ -222,6 +225,8 @@ def getMotor(self, name: str) -> Motor: ...
     def getReceiver(self, name: str) -> Receiver: ...
     def getTouchSensor(self, name: str) -> TouchSensor: ...
 
+    def getDevice(self, name: str) -> Optional[TDevice]: ...
```

Comment:
I'm surprised this works? I think this should just be `Device` rather than `TDevice`. For generic parameters in return types there generally needs to be another mention of the same type parameter within the types of the arguments -- so that you can know what the concrete type is when the function is called.